crypto/ec/ecp_nistz256.c

```diff
@@ -390,6 +390,9 @@ static void ecp_nistz256_point_add(P256_POINT *r,
      *
      * We use bitwise operations to avoid potential side-channels introduced by
      * the short-circuiting behaviour of boolean operators.
+     *
+     * This case should never happen during single point multiplication, so
+     * there is no timing leak for ECDH or ECDSA signing.
      */
     if (is_equal(U1, U2) & ~in1infty & ~in2infty & is_equal(S1, S2)) {
         ecp_nistz256_point_double(r, a);
```

Comment:
I think you need a similar comment as the p521 here, since otherwise the above comment
could be unclear which case exactly is meant?

Reply:
Done, for all the changed `.c` files, I moved the constant-timeness comment inside the relevant `if` branch.